Keyboards/LanguageKeyboards/Hebrew/HEInterfaceVariables.swift

```diff
@@ -1,51 +1,32 @@
-/**
```

Comment:
Can we get this file header put back in, @DanielGevaZA? :) Is needed for the license.